sdk/messaging/azeventhubs/internal/namespace_test.go

```diff
@@ -118,7 +118,7 @@ func TestNamespaceNegotiateClaimRenewal(t *testing.T) {
 	}
 
 	var errorsLogged []error
-	var nextRefreshDurationChecks atomic.Int32
+	var nextRefreshDurationChecks int32
```

Comment:
Oh, I know this "downgrade" feeling. They added some nice stuff in atomics in 1.19.

Reply:
Yeah I was a bit bummed about having to make this switch.